samples/unit-tests/series/datagrouping/demo.js

```diff
@@ -827,6 +827,97 @@ QUnit.test('When groupAll: true, group point should have the same start regardle
     );
 });
 
+QUnit.test('When groupAll: true, group point should have the same start regardless of axis extremes (case with cropStart), #15005.', function (assert) {
```

Comment:
Can we reuse the previous demo for this extra use-case? The demo is defined a few lines above new one